src/pages/_app.tsx

```diff
@@ -21,11 +21,6 @@ function MyApp({ Component, pageProps: { session, ...pageProps }, }: AppProps<Cu
     <Provider store={store}>
       <SessionProvider session={session}>
         {/*<Navbar />*/}
-        <Head>
```

Comment:
was getting complaints from next about putting stylesheets in _app file, so moved to new _document file: https://nextjs.org/docs/messages/no-stylesheets-in-head-component